### PR TITLE
fix: added instructions for npm link and storybook

### PR DESCRIPTION
**Pull request by chavlji**
Readme update

### CODE REVIEW

**Comment by zenbot-0** [comment]
🔬 [Click here](https://reciprocity.github.io/zenlib/branches/quickfix-update-readme) to test your changes on Storybook

File: README.md [hunk 1]
```diff
@@ -47,25 +47,25 @@ _The storybook website will also watch for changes while it's running. So if you
...
-`cd zenlib/packages/multiselect`\
-...and create a symlink to make component globally available:\
+Go to `zenlib/packages/my-component` and create a symlink to make folder globally available:\
 `yarn link`
 
-Go to the zengrc\frontend folder:\
-`cd zengrc/frontend`\
-... and substitute actual module with a symlink:\
-`yarn @reciprocity/multiselect`
+Go to `zengrc\frontend` folder and substitute actual module with a symlink:\
```
Comment (leferreyra) [inline README.md:61]: This is done by `yarn link` though, you don't need to do it manually.

File: README.md [hunk 2]
```diff
@@ -47,25 +47,25 @@ _The storybook website will also watch for changes while it's running. So if you
 
 #### Reiewing component changes live in the Zengrc app
```
Comment (leferreyra) [inline README.md:48]:
```suggestion
#### Reviewing component changes live in the Zengrc app
```

**Comment by leferreyra** [comment]
Just as an FYI, there's a docs category for commits like these.
https://www.conventionalcommits.org/en/v1.0.0/

File: README.md [hunk 3]
```diff
@@ -47,25 +47,25 @@ _The storybook website will also watch for changes while it's running. So if you
...
-...and create a symlink to make component globally available:\
+Go to `zenlib/packages/my-component` and create a symlink to make folder globally available:\
 `yarn link`
 
-Go to the zengrc\frontend folder:\
-`cd zengrc/frontend`\
-... and substitute actual module with a symlink:\
-`yarn @reciprocity/multiselect`
+Go to `zengrc\frontend` folder and substitute actual module with a symlink:\
+`yarn @reciprocity/my-component`
```
Comment (mplesec) [inline README.md:62]: I think this should be `yarn link @reciprocity/my-component`.

**Comment by chavlji** [comment]
@mplesec suggestions are always welcome.

**Comment by chavlji** [comment]
Closing due to failing checks (github issue). I'll create a new pr.

**Pull request closed without merging**